### ISSUE TITLE
[crash][1.12]Immersive Engineering Cloche+EXU2 transfer node crash

### ISSUE THREAD

**Issue by Buggy123** [BluSunrize/ImmersiveEngineering#2560]
Immersive Engineering version 0.74
Extra Utilities version "${version} (1.0)"
Minecraft 1.12.2
Modpack: All The Mods 3 v4.2 singleplayer

Not sure if this is a EXU2 problem or a immersive engineering problem, but there's a issue with garden cloches and transfer pipes/nodes which causes a crash on world load.

I have a setup where a transfer node is horizontally adjacent to, but not extracting from, the top block of a cloche (which doesn't transfer or receive items anyway). Upon loading the world, this crashes. Removing the transfer node via MCEdit stops the crash, and replacing the transfer node doesn't cause a crash until the world is reloaded again.

Crash report: https://pastebin.com/343nqDr3

I've submittted this to both the EXU2 and IE githubs.

**Comment by BluSunrize** [comment]
@malte0811, we've been getting multiple reports of these, any chance they were caused by the commit changing how States are handled?
I didn't get to looking at it closer yet, mostly guesstimating.

**Comment by malte0811** [comment]
It shouldn't but probably is. Aka I don't know why it would be as the code I added should run before the old code setting the direction runs, but it is the only commit I can think of that is somewhat related.
I'll try to take a look later today, unless you fix it/assign yourself first.

**Comment by malte0811** [comment]
This one specifically was caused by my new code setting the cloche as facing down (the default value of the FACING_ALL property, this would have been corrected when loading from NBT) and XU asking for capabilities on the client side before the sync packets were processed. The stripcurtain and the auto workbench crashed for similar reasons (asking for a bounding box before the NBT was loaded properly).

**Comment by prozacgod** [comment]
I had a similar crash and noticed another nuance to the bug.... MAYBE it's a new one or not... 

The crash/break error
https://paste.ee/p/UjlCg

BUT the top of my cloche can rotate independant of the bottom... at least in the way it presents it'self to transfer devices (pipes and such).

I had an existing item duct with a water pipe below and placed a cloche on the water pipe, It was not connected to the item duct so I rotated it.  By clicking on the base.

In no orientation did it ever seem to connect to the pipe, but I ended up whacking the bell jar and while the render of the cloche never changed, the connection to the duct did.

Odd... 

Does this fix also address that?
